lib/hammer_cli_foreman/commands.rb

```diff
@@ -744,7 +744,8 @@ def execute
       response = send_request
       if option_path
         filepath = store_response(response)
-        print_message(_('The response has been saved to %{path}s.'), {:path => filepath})
+        source = resource.nil? ? "response" : resource.singular_name.gsub('_', ' ')
+        print_message(_('The %{source} has been saved to %{path}s.'), {:path => filepath, :source => source})
```

Comment:
This is not quite what I'd expect :) For English only it will just work, but let's take a different language:

In French the previous string will be translated into 
> La réponse a été enregistrée dans %{path}s.

But with this change it could be

> La response a été enregistrée dans %{path}s.

Although we could extract this word (`_('response')`), we still would put there English version of the resource, not translated one, so in case of report templates it would be

> La report template a été enregistrée dans %{path}s.

FYI, I've just noticed that we have no translated strings in the repo for quite a long time, although they are present in transifex :facepalm: 

P.S. There is redundant `s` in the end of the message for some reason...

Reply:
I am not sure I followed. , why does this solution won't work? 
``` print_message(_('The %{source} has been saved to %{path}.'), {:path => filepath, :source => _(source)})```